lib/template/spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 #
-ENV["RACK_ENV"] = "test"
+ENV["PLINY_ENV"] = "test"
```

Comment:
This does go on to reinforce your point though :) Sinatra implicitly modifying its behavior based on `RACK_ENV` is incredibly confusing and very bad.